README.md

```diff
@@ -83,6 +116,247 @@ has instructions for running the samples.
 | --------------------------- | --------------------------------- |
 | Examples | [source code](https://github.com/googleapis/nodejs-error-reporting/blob/master/samples/snippets.js) |
 
+## Configuration
+
+The following code snippet lists all available configuration options.  All configuration options are optional.
```

Comment:
`all` is probably too strong. I think there are other options that the `Service` parent class accepts that we simply pass on.

Reply:
Good catch.